app/views/curation_concerns/base/inspect_work.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1><%= t("inspect_work.link_text") %></h1>
+<ul class="breadcrumb">
+  <li>
+    Back to <%= link_to @presenter.to_s, [main_app, @presenter] %>
+  </li>
+</ul>
+
+<h3>Workflow</h3>
+<dl>
+  <% workflow = @presenter.inspect_work.workflow %>
+  <dt>Object Name</dt>
+  <dd><%= @presenter.to_s %></dd>
+  <dt>Processing Entity ID</dt>
+  <dd><%= workflow[:entity_id] %> (Proxy for <%= workflow[:proxy_for] %>)</dd>
+  <dt>Workflow Name</dt>
+  <dd><%= workflow[:workflow_name] %> (ID=<%= workflow[:workflow_id] %>)</dd>
+  <dt>Workflow State</dt>
+  <dd><%= workflow[:state_name] %> (ID=<%= workflow[:state_id] %>)</dd>
+  <dt>Workflow Comments</dt>
+  <% workflow[:comments].each do |comment| %>
+      <dd><%= comment[:created_at] %>, <%= comment[:comment] %></dd>
+  <% end %>
+  <dt>Roles</dt>
+  <dd><%= workflow[:roles] %></dd>
+</dl>
+
+<h3>Storage</h3>
+<dl>
+  <dt>Work persistence</dt>
+  <dd><%= @presenter.inspect_work.storage %></dd>
```

Comment:
I would just leave this part out, it's slow and unlikely to give you anything useful that the solr version wouldn't